File: src/main/java/me/mixces/animatium/util/MathUtils.java
package me.mixces.animatium.util;

public abstract class MathUtils {
    public static float toRadians(float angle) {
        return angle * (float) Math.PI / 180F;
    }

    public static float toAngle(float radians) {
        return radians / (float) Math.PI * 180F;
    }
}

File: src/main/kotlin/me/mixces/animatium/util/MathUtils.kt
package me.mixces.animatium.util

abstract class MathUtils {
    companion object {
        fun toRadians(angle: Float): Float {
            return angle * Math.PI.toFloat() / 180F
        }

        fun toAngle(radians: Float): Float {
            return radians / Math.PI.toFloat() * 180F
        }
    }
}